### ISSUE TITLE
Implement Recently Closed Tabs

### ISSUE THREAD

**Issue by PalmerAL** [PalmerAL/min#3]

**Comment by PalmerAL** [comment]
This should also allow for groups of closed tabs - for example, when pressing mod+n.

**Comment by andrewjwilson** [comment]
Yes, this is a must have feature for me too... my fingers are often too fast in closing down tabs!

**Comment by mardukbp** [comment]
+1 This feature is always handy.
